### PR TITLE
chore(bigquery): Add geography samples

### PR DESCRIPTION
**Pull request by quartzmo**
* Add bigquery_insert_geography_wkt
* Add bigquery_insert_geojson
* Update Rakefile to Bundler.with_unbundled_env

closes: #13295

### CODE REVIEW

**Review by shollyman — APPROVED** [review]
Thanks!  LGTM, modulo the one test failure that seems to be whitespace formatting related.
```
google-cloud-bigquery: rubocop ...
Running RuboCop...
RuboCop failed!
Inspecting 92 files
......................................C.....................................................

Offenses:

lib/google/cloud/bigquery/service.rb:604:46: C: [Correctable] Layout/LeadingCommentSpace: Missing space after #.
          def retry? result, current_retries #:nodoc:
                                             ^^^^^^^^

92 files inspected, 1 offense detected, 1 offense auto-correctable
```

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  dataset  = bigquery.dataset dataset_id
+  table    = dataset.table table_id
+
+  # Use the RGeo library to generate WKT of a line from LAX to
+  # JFK airports. Alternatively, you may define WKT data directly.
+  factory = RGeo::Cartesian.factory
+  my_geography = factory.line_string([factory.point(-118.4085, 33.9416), factory.point(-73.7781, 40.6413)])
+  row_data = [
+    # Convert data into a WKT string: "LINESTRING (-118.4085 33.9416, -73.7781 40.6413)"
+    { geo: my_geography.to_s }
```
Comment (dazuma) [inline google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb:30]: Use `.as_text` rather than `.to_s`.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+def insert_geography_wkt dataset_id = "your_dataset_id", table_id = "your_table_id"
+  bigquery = Google::Cloud::Bigquery.new
+  dataset  = bigquery.dataset dataset_id
+  table    = dataset.table table_id
+
+  # Use the RGeo library to generate WKT of a line from LAX to
+  # JFK airports. Alternatively, you may define WKT data directly.
+  factory = RGeo::Cartesian.factory
+  my_geography = factory.line_string([factory.point(-118.4085, 33.9416), factory.point(-73.7781, 40.6413)])
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb:27]: Trying to remain consistent with the Python samples :(

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb (hunk 1 above)
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb:30]: Thanks! I was looking for the best way to do this but didn't find `as_text`.

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+require "rgeo"
+
+def insert_geography_wkt dataset_id = "your_dataset_id", table_id = "your_table_id"
+  bigquery = Google::Cloud::Bigquery.new
+  dataset  = bigquery.dataset dataset_id
+  table    = dataset.table table_id
+
+  # Use the RGeo library to generate WKT of a line from LAX to
+  # JFK airports. Alternatively, you may define WKT data directly.
+  factory = RGeo::Cartesian.factory
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb]: Done, thanks!

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb]: Done, thanks!

File: google-cloud-bigquery/samples/snippets/insert_geojson.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+def insert_geojson dataset_id = "your_dataset_id", table_id = "your_table_id"
+  bigquery = Google::Cloud::Bigquery.new
+  dataset  = bigquery.dataset dataset_id
+  table    = dataset.table table_id
+
+  # Use the RGeo library to generate GeoJSON of a line from LAX to
+  # JFK airports. Alternatively, you may define GeoJSON data directly, but it
+  # must be converted to a string before loading it into BigQuery.
+  factory = RGeo::Cartesian.factory
+  my_geography = factory.line_string([factory.point(-118.4085, 33.9416), factory.point(-73.7781, 40.6413)])
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geojson.rb]: Done, thanks!

File: google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb (hunk 1 above)
Comment (quartzmo) [reply google-cloud-bigquery/samples/snippets/insert_geography_wkt.rb]: Done, thanks!

**Review by dazuma — APPROVED** [review]
LGTM!

**Pull request merged**